components/script/script_module.rs

```diff
@@ -1434,18 +1501,26 @@ fn fetch_an_import_module_script_graph(
     global: &GlobalScope,
```

Comment:
I didn't find anything at the `html` link, but I did come across https://tc39.es/ecma262/#sec-HostLoadImportedModule, which I think is how this operation is specified?